renku/cli/run.py

```diff
@@ -208,6 +213,8 @@
 
 @click.command(context_settings=dict(ignore_unknown_options=True,))
 @click.option("--name", help="A name for the workflow step.")
+@click.option("--description", help="Workflow step's description.")
+@click.option("--keyword", multiple=True, help="List of tags for the workflow.")
```

Comment:
We should not say "tags" in the description, "keywords" and "tags" is already confusing for users on datasets, and the distinctions isn't always clear. so we be as consistent as possible on our end.